back/app/controllers/web_api/v1/phases_controller.rb

```diff
@@ -10,11 +10,11 @@ def index
       .order(:start_at)
     @phases = paginate @phases
 
-    render json: linked_json(@phases, WebApi::V1::PhaseSerializer, params: jsonapi_serializer_params)
+    render json: linked_json(@phases, WebApi::V1::PhaseSerializer, params: jsonapi_serializer_params, include: %i[permissions])
```

Comment:
@sebastienhoorens Just made this change to Amanda's PR to include the permissions with the phases when they are requested. However, the permissions relationship in the serializer is added by the granular permissions engine. Would it be better to patch the engine - even though as far as I'm aware there are no tenants that don't have granular permissions enabled.